modules/katello_activation_key.py

```diff
@@ -201,6 +211,25 @@ def activation_key(self, name, organization, lifecycle_environment=None, content
 
         return updated
 
+    def activation_key_copy(self, name, organization, new_name):
+        updated = False
+        organization = self.find_organization(organization)
+
+        kwargs = {'name': name, 'organization': organization}
+        activation_key = self._entities.ActivationKey(self._server, **kwargs)
+        response = activation_key.search({'name', 'organization'})
+
+        kwargs = {'name': new_name, 'organization': organization}
+        new_activation_key = self._entities.ActivationKey(self._server, **kwargs)
+        new_response = new_activation_key.search({'name', 'organization'})
+
+        if len(response) == 1 and len(new_response) == 0:
+            if not self._module.check_mode:
+                new_activation_key = response[0].copy(data={'new_name': new_name})
```

Comment:
I think adding an `elif state == 'copied':` to https://github.com/theforeman/foreman-ansible-modules/blob/2dedb388bb94bd17afd80325c365e326fda254f3/module_utils/ansible_nailgun_cement.py#L237 and using that would be useful for content_views as well.